llvm_passes/CustomTensorOperatorInstSelector.cpp

```diff
@@ -96,8 +78,10 @@ class CustomTensorOperatorInstSelector : public HardwareFIInstSelector {
 
             strcpy(opname, name.c_str());
 
+            std::cout<<"OpertaorName: "<<opname<<"\n";
```

Comment:
Can this be changed to "OperatorName" to fix the typo?

Reply:
Yes. Pushed this change in the latest commit (234e1b06b4731ab62518840a10bf28c083e934f4).